src/common/image.c

```diff
@@ -2735,6 +2736,19 @@ float dt_image_get_exposure_bias(const struct dt_image_t *image_storage)
     return 0.0f;
 }
 
+void dt_image_check_camera_missing_sample(const struct dt_image_t *img)
+{
+  if(img->camera_missing_sample)
+  {
+    dt_control_log
+      (_("<big><span foreground='red'><b>ATTENTION</b> : camera is missing samples</span>\n\n"
+         "You must provide samples in <a href='https://raw.pixls.us/'>https://raw.pixls.us/</a> for:\n\n"
+         "`%s' `%s' (compressed/uncompressed/all bit depths)\n\n"
```

Comment:
maybe we can precise something like "in as many format/compression/bit depths as possible" to avoid discouraging people who don't own the camera anymore to provide all the sample they have (usually only 1 format, the one they used)

That's also where a dedicated page could help (but that's something to do/discuss later)

Reply:
Done.